.travis.yml

```diff
@@ -1,12 +1,41 @@
+conditions: v1
 language: python
+
 python:
-  - "2.6"
-  - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-  - "3.5-dev" # 3.5 development branch
-  - "nightly" # currently points to 3.6-dev
-# command to run tests
+- 2.6
+- 2.7
+- 3.2
+- 3.3
+- 3.4
+- 3.5
+- 3.6
+- 3.7
```

Comment:
Look at https://travis-ci.org/bosswissam/pysize/builds/443226261. There's two jobs for 3.7. The first one (failing) is this line and the second one is one in `jobs.include`, which is configured properly.

The way it works is that Travis CI generates a matrix of jobs from `python:` section and for each of them it uses `script:`/`install:` from the root level of this config.

Jobs in `jobs.include` inherit all of the steps from the root level, but when you explicitly specify different values for things those override inherited settings.

So for the failing job the config is:
```yaml
dist: trusty  # implicit default
sudo: false  # implicit default
language: python  # specified explicitly
python: 3.7  # specified explicitly
install: pip install -r requirements.txt  # implicit default
script: nosetests  # specified explicitly
```
And for succeeding job it's:
```yaml
dist: xenial  # specified explicitly, overrides value from config root level
sudo: true  # specified explicitly, overrides value from config root level
python: 3.7  # specified explicitly, overrides value from config root level
install: pip install -r requirements.txt  # inherited from config root
script: nosetests  # inherited from config root
```

P.S. If you are not sure how the config you wrote has been parsed by Travis, there's a "View config" tab in each job, like:
- https://travis-ci.org/bosswissam/pysize/jobs/443226269/config
- https://travis-ci.org/bosswissam/pysize/jobs/443226270/config